wordpress-mobile/WordPress-Android
# Site Settings: Categories - Create skeleton activity, fragment, vm, and vm test

Parent #5382 

This PR creates skeleton activity, fragment, vm, and vm test for the Site Settings: Categories feature.

To test:
- Go to Me -> App Settings -> Test Feature Configuration
- Enable ManageCategoriesFeatureConfig and restart the app
- When the app relaunches, navigate back to My Site -> Settings
- Tap on the Categories row (in the Writing section)
- Note that the Categories view is shown

## Regression Notes
1. Potential unintended areas of impact 🟢 
2. What I did to test those areas of impact (or what existing automated tests I relied on) 🟢 
3. What automated tests I added (or what prevented me from doing so) 🟢 

PR submission checklist:
